pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -112,12 +112,8 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		}
 		glog.Infof("[cpumanager] detected CPU topology: %v", topo)
 		reservedCPUs, ok := nodeAllocatableReservation[v1.ResourceCPU]
-		if !ok {
-			// The static policy cannot initialize without this information.
-			return nil, fmt.Errorf("[cpumanager] unable to determine reserved CPU resources for static policy")
-		}
-		if reservedCPUs.IsZero() {
-			// The static policy requires this to be nonzero. Zero CPU reservation
+		if !ok || reservedCPUs.IsZero() {
+			// The static policy requires reserved CPUs to be nonzero. Zero CPU reservation
```

Comment:
i am not sure if combining two error scenarios into a single error scenario is the right thing to do here

Reply:
Well the second scenario never happens (at least not in the system I was testing). If we specify 0 reserved cpus then there is no entry for the key ResourceCPU in the nodeAllocatableReservation. And the same case happens when we do not specifically specify any CPUs.